### PR TITLE
Dev/rename to symon

### PR DESCRIPTION
**Pull request by trottier**
Performed rename of Simon to Symon.

### CODE REVIEW

File: games/017_Symon/project.properties [hunk 1]
```diff
@@ -0,0 +1,2 @@
+name=Symon
```
Comment (JelmerT) [inline games/017_Symon/project.properties]:
```suggestion
name=symon
```
needs to match the filename

File: games.json [hunk 2]
```diff
@@ -148,25 +148,17 @@
 		"author": "CleverPet <hello@clever.pet>",
 		"description_sentence":"Once your dog or cat can see the lights easily, this is a more \"active\" game for the Hub that challenges your dog or cat's \"paw-eye\" coordination",
 		"description_paragraph":"",
+		"type": "github.com",
```
Comment (JelmerT) [inline games.json]:
```suggestion
```
This shouldnt be here

**Comment by JelmerT** [comment]
Something weird happened somewhere, best to rebase your commits on top of the current master.

Some file names were renamed back to 016 instead of 017, and some changes in the current master were reverted.

**Comment by JelmerT** [comment]
Rebased, fixed all the issues and added a standard readme for the Symon game.

**Comment by trottier** [comment]
Yes, I've fixed my current master so this won't happen again.

**Pull request merged**